site/app/controllers/admin/PlagiarismController.php

```diff
@@ -296,7 +296,7 @@ public function saveNewPlagiarismConfiguration($new_or_edit, $gradeable_id = nul
             $this->core->redirect($return_url);
         }
 
-        if (isset($_POST['sequence_length']) && $_POST['sequence_length'] !== '' && is_numeric($_POST['sequence_length']) && $_POST['sequence_length'] > 0) {
+        if (isset($_POST['sequence_length']) && $_POST['sequence_length'] !== '' && is_numeric($_POST['sequence_length']) && $_POST['sequence_length'] >= 2) {
             $sequence_length = $_POST['sequence_length'];
         }
         else {
```

Comment:
Could we maybe add the minimum number required for the sequence length to the error message (line 303)?
Something like:
```php
$this->core->addErrorMessage("Invalid input provided for sequence length. The minimum threshold value is 2");